strr-examiner-web/app/pages/examine/[applicationId].vue

```diff
@@ -96,16 +114,19 @@ watch(
     initialMount.value = false
     updateRouteAndButtons(RoutesE.EXAMINE, {
       approve: {
-        action: (id: string) => handleApplicationAction(id, ApplicationActionsE.APPROVE, 'right', 1),
-        label: t('btn.approve')
+        action: (id: string) => handleAssigneeAction(id, ApplicationActionsE.APPROVE, 'right', 1),
```

Comment:
Nested functions could be reduced to just checking `isAssignedToUser.value` and calling original `handleApplicationAction()` or open confirm modal.

Reply:
I will address these comments during next sprint.